modules/identityserver/src/Volo.Abp.IdentityServer.Domain/Volo/Abp/IdentityServer/Clients/IClientCorsOriginRepository.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Volo.Abp.Domain.Repositories;
+using Volo.Abp.IdentityServer.Clients;
+
+namespace Volo.Abp.IdentityServer
+{
+    public interface IClientCorsOriginRepository:IBasicRepository<ClientCorsOrigin>
```

Comment:
We should not define repositories for non-aggregate root entities because they are not collections in the MongoDB, hence will not work.